apps/protocol-frontend/src/components/AttestationsTableShell.tsx

```diff
@@ -28,6 +29,7 @@ const AttestationsTableShell = () => {
     hasNextPage,
     fetchNextPage,
   } = useContributionInfiniteList({
+    orderBy: { date_of_engagement: SortOrder.Desc },
```

Comment:
Awesome. Having this on the orderBy on the query is great and is now behaving as users would expect 🔥